kubernetes/charts/direktiv/values.yaml

```diff
@@ -60,6 +60,11 @@ flow:
   image: "vorteil/flow"
   tag: ""
   db: ""
+  prometheus:
```

Comment:
I would not put that under flow but have it as a stand-alone entity in values.yaml

Reply:
`prometheus` is now it's own top-level object in the default values file.